res/app/control-panes/video-recorder/video-recorder-spec.js

```diff
@@ -0,0 +1,17 @@
+describe('VideoRecorderCtrl', function() {
+
+  beforeEach(angular.mock.module(require('.').name))
+
+  var scope, ctrl
+
+  beforeEach(inject(function($rootScope, $controller) {
+    scope = $rootScope.$new()
+    ctrl = $controller('VideoRecorderCtrl', {$scope: scope})
+  }))
+
+  it('should ...', inject(function() {
+    expect(1).toEqual(1)
```

Comment:
It looks like autogenerated test stub.

Reply:
i just copy it from screenshots folder, yeah looks like it's auto generated file, i just thought it's kind of reminder to create a test, should i get rid of it?

Reply:
Yep, of course it is better to write a proper test if it is possible.